.expeditor/scripts/shared.sh

```diff
@@ -40,6 +40,22 @@ curlbash_hab() {
     # are the same. let's just delete it
     rm -rf /hab/pkgs/core/hab/0.82.0
     curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh | sudo bash -s -- -t "$pkg_target"
+    case "${pkg_target}" in
+        x86_64-linux)
+            hab_binary="/bin/hab"
+            ;;
+        x86_64-linux-kernel2)
+            hab_binary="/bin/hab"
+            ;;
+        x86_64-darwin)
+            hab_binary="/usr/local/bin/hab"
+            ;;
+        *)
+            echo "--- :no_entry_sign: Unknown PackageTarget: ${pkg_target}"
+            exit 1
+            ;;
+    esac
+    declare -g hab_binary
```

Comment:
I'm still not sure about this... the whole idea of the curlbash install is that you get `hab` on your path.